src/extension/report.ts

```diff
@@ -0,0 +1,50 @@
+import * as Sentry from "@sentry/node";
+import { v4 as uuidv4 } from "uuid";
+import {
+    workspace,
+} from "vscode";
+
+export class CrashReporter {
+    private enableCrashReporting: any;
+
+    constructor(
+        enableCrashReporting: boolean,
```

Comment:
not sure we need anything passed into the constructor as we always use `telemetry` to figure out if we want to activate crash reporting 🤔

Reply:
Ok. That is right